### DISCUSSION TITLE
Tutanota - Services like in the primitive era!

### DISCUSSION THREAD

**Discussion by addison74** [tutao/tutanota#5765] [General]
On one of the servers I manage, a user reported to me that messages sent from an address @tutanota.com are not received. I checked the logs of the email server (postfix) and there is no connection with the tutanota server. So I went further with the investigation and I created an email address and indeed the messages are not sent. After a while there is an attachment with an error related to port 25.

**Question**: If port 25 is closed or does not offer TLS, why doesn't the Tutanota server also try on port 587?

I sent several emails for testing and now what to see on my screen? I violated the terms and conditions and the account can no longer be used.

**Question**: Suspending an account without having the opportunity to justify yourself seems to me lacking in any professional ethics. More than that, it is scandalous that there is no email address or contact form where anyone can complain about not receiving messages from their server, about issues.

There is still a long way to go for the gentlemen from Tutanota.

**Comment by charlag** [comment]
You should contact hello@tutao.de for all of these issues

**Comment by addison74** [comment]
@charlag - I followed your advice and I sent a message to that email address explaining the situation in detail. A week has passed and I have not received any reply. The account is suspended. I have never encountered a similar situation with another email provider. 

This post comes to confirm what I said before, without professional support in the relationship with customers, Tutanota does nothing but create problems for those who want to use their services. After this unpleasant experience I will not recommend Tutanota and I will create a banner in the online stores that I manage to inform the visitors not using Tutanota email addresses for contact.
